File: front/amadeusSports/src/app/dto/BetDto.ts
export class BetDto {
    id!: number;
    time!: string;
    team1!: string;
    team2!: string;
    score!: string;
    sport!: string;
    league!: string;
    amount!: number;
    status!: string;
    result!: number;
    constructor(id: number, time: string, team1: string, team2: string, score: string, sport: string, league: string,
        amount: number, status: string, result: number) {
        this.time = time;
        this.amount = amount;
        this.league = league;
        this.sport = sport;
        this.team1 = team1;
        this.team2 = team2;
        this.score = score;
        this.status = status;
        this.result = result;
    }
}

File: back/src/main/java/com/example/back/controllers/dto/BetDto.java
package com.example.back.controllers.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class BetDto {
    private Long id;
    private String time;
    private TeamDto team1;
    private TeamDto team2;
    private String score;
    private String sport;
    private LeagueDto league;
    private int amount;
    private String status;
    private int result;
    private UserBetChoiceDto betChoice;
}
